hic_qc.py

```diff
@@ -648,13 +688,13 @@ def plot_histograms(self):
                                       np.log10(max_dist),
                                       50),
                      log=True, edgecolor='black', color='red', normed=True)
-            ax.set_ylim(0.5, max(num_pairs * 2, 1))
+            ax.set_ylim(0.0, 1.0)
             plt.yscale('log', nonposy='clip')
             plt.xscale('log')
             plt.xlim(xmin=1)
             plt.title(title_string)
             plt.xlabel('Distance between read pair mates in Hi-C mapping (same contig, log scale)')
-            plt.ylabel('Density of reads (log scale)')
+            plt.ylabel('Density of reads (percentage, log scale)')
```

Comment:
It doesn't look like a percentage based on the example report.

Reply:
They sum to 1. Would "density" be more clear?

Reply:
Yeah

Reply:
Changed to density